addons/main/script_macros_common.hpp

```diff
@@ -796,6 +796,17 @@ Macro: GVARMAIN()
     #define PREPMAIN(var1) ['PATHTO_SYS(PREFIX,COMPONENT_F,DOUBLES(fnc,var1))', 'TRIPLES(PREFIX,fnc,var1)'] call SLX_XEH_COMPILE_NEW
 #endif
 
+#ifdef RECOMPILE
+    #define RECOMPILE recompile = 1
```

Comment:
does this work with Mikero's Tools? I think they error out now when trying to redefine a macro in config space.